monitoring/charts/prometheus-collector/templates/config.yaml

```diff
@@ -9,12 +9,27 @@ data:
     aggregated:container_cpu_usage_seconds_total{} = sum(container_cpu_usage_seconds_total) by (id) keep_common
     aggregated:container_cpu_user_seconds_total{} = sum(container_cpu_user_seconds_total) by (id) keep_common
 
+  alert.rules: |
```

Comment:
Alerting rules are configured and evaluated in the prometheus server. Subsequently alerts are sent to the alertmanager.